influxdb_client/client/tasks_api.py

```diff
@@ -65,7 +64,11 @@ def _create_task(name: str, flux: str, every, cron, org_id: str) -> Task:
             repetition += "cron: "
             repetition += '"' + cron + '"'
 
-        flux_with_options = 'option task = {{name: "{}", {}}} \n {}'.format(name, repetition, flux)
+        from_index = flux.index("from")
```

Comment:
Hmm... I'm a little concerned that this doesn't quite cover the use case when the user sets the entire query to a variable.
For example:
```
base_query = from(....

base_query |> stuff(...
base_query |> more_stuff(...
```

Reply:
Wonder if you have to resort to splitting the flux script by line and inserting the options before the line instead of index itself.

Reply:
Or technically, can't "option task ..." be appended at the very end of the flux script?

Reply:
> Or technically, can't "option task ..." be appended at the very end of the flux script?

Good point. The `option task` can be appended at the end of the script. This is valid task definition:

```flux
import "http"

from(bucket: "iot_center")
    |> range(start: -30d)
    |> filter(fn: (r) => r._measurement == "environment")
    |> aggregateWindow(every: 1h, fn: mean) 
    
 option task = {name: "it task1660540801.907463-IT", cron: "10 0 * * * *"}
```